spec/requests/api/v1/users_unify_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe Api::V1::UsersController do
+  let(:no_headers) { {'HTTP_ACCEPT': 'application/json'} }
+
+  describe 'GET /api/v1/unify' do
+    let(:user_1) { FactoryGirl.create(:user, user_name: 'Thomas', mentor: true) }
```

Comment:
Okay. 
Can you explain why and where we should use FactoryGirl?
e.g. could we refactor request specs for `users_skills` and `users_friendship` then as well to use just `.create`?